tests/test_opds_import.py

```diff
@@ -519,10 +519,10 @@ def test_resources_are_mirrored_on_import(self):
         # Each resource was 'mirrored' to an Amazon S3 bucket.
         # The first resource has no bibframe provider in OPDS so it uses the importer's data source.
         eq_(
-            ['http://s3.amazonaws.com/test.content.bucket/Library%20Simplified%20Open%20Access%20Content%20Server/Gutenberg%20ID/10557.epub',
-             'http://s3.amazonaws.com/test.cover.bucket/Gutenberg/Gutenberg%20ID/10441/cover_10441_9.png.svg', 
+            ['http://s3.amazonaws.com/test.content.bucket/Library%20Simplified%20Open%20Access%20Content%20Server/Gutenberg%20ID/10557/Johnny%20Crow%27s%20Party.epub.images',
+             'http://s3.amazonaws.com/test.cover.bucket/Gutenberg/Gutenberg%20ID/10441/cover_10441_9.png', 
```

Comment:
I'm not sure I like the way the changes affect this case. I'm preserving the original extension to handle something like "epub3" where multiple extensions can match a media type. But here the existing extension matches _a different extension_.